packages/react-button/docs/button-demo.stories.js

```diff
@@ -15,46 +15,47 @@ export const demo = (args) => <Button {...args} />;
 demo.storyName = 'Demo';
 demo.args = {
 	label: 'Button Label',
+	design: 'solid',
+	color: 'gray',
+	loading: false,
+	disabled: false,
+	iconRight: false
 };
 demo.argTypes = {
 	label: {
 		description: 'The text that shows up in the button',
 		table: {
-			type: { summary: 'string' },
-			defaultValue: { summary: '' },
```

Comment:
@creador-dev, `defaultValue` summary is still supported. Please, read through it.